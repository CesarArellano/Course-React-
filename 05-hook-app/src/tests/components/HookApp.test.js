import { shallow } from "enzyme";
import { HookApp } from "../../HookApp";

describe('Testing HookApp Component', () => {
  const wrapper = shallow(<HookApp />);
  
  test('Must show component correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
  
})
