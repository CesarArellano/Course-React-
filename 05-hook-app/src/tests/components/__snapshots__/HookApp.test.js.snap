// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing HookApp Component Must show component correctly 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
